### PR TITLE
[AC-7898] Add two fields to office_hours_calendar output

### PR DESCRIPTION
**Pull request by jonkiparsky**
https://masschallenge.atlassian.net/browse/AC-7898

To test: create an office hour slot for some mentor and reserve it for some finalist, selecting a startup. Check the office_hours_calendar api endpoint for the mentor. startup's short_pitch and primary industry should be included in the output.

### CODE REVIEW

File: web/impact/templates/emails/reserve_office_hour_email_to_finalist.html [hunk 1]
```diff
@@ -23,6 +23,12 @@
       Yahoo
     </a>
   </li>
+  <li>
```
Comment (iankigen) [inline web/impact/templates/emails/reserve_office_hour_email_to_finalist.html]: This work is not part of def of done. Is there any reason why you included it as part of this PR?

**Review by iankigen — COMMENTED** [review]
Code/QA looks good:

- The endpoint office_hours_calendar includes the following fields in calendar_data:
    - startup_short_pitch ✅ 
     - startup_primary_industry ✅

File: web/impact/templates/emails/reserve_office_hour_email_to_finalist.html (hunk 1 above)
Comment (jonkiparsky) [reply web/impact/templates/emails/reserve_office_hour_email_to_finalist.html]: Looks like some sort of merge damage. Removing

**Review by iankigen — APPROVED** [review]

**Review by geofrocker — APPROVED** [review]
Looks good!

**Pull request merged**